polictf-2015/pwnable/johns-library/README.md

```diff
@@ -18,5 +18,6 @@
 
 ## Other write-ups and resources
 
+* <https://github.com/thebarbershopper/ctf-writeups/tree/master/polictf-2015/johns-library>
```

Comment:
@thebarbershopper the preference will be first come first serve only any way I will move them it to the bottom :smile: